sycl/unittests/pi/BackendString.hpp

```diff
@@ -0,0 +1,23 @@
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
The header seems to be a little different from what the rest of the project uses.